.gitmodules

```diff
@@ -79,3 +79,6 @@
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
 	url = https://github.com/onnx/onnx-tensorrt.git
+[submodule "cmake/external/tvm_update"]
+	path = cmake/external/tvm_update
+	url = https://github.com/jroesch/tvm
```

Comment:
Is it possible to have this track the official branch of TVM, even if on the same SHA?

Reply:
I thought It has been changed automatically when I pulled TVM from origin repo, but github deceived my expectations. I've fixed url in .gitmodules